src/Data/YAML/Token.hs

```diff
@@ -1482,9 +1482,9 @@ ns_flow_map_explicit_entry n c {- 143 -} = ns_flow_map_implicit_entry n c
                                            & e_node )
 
 ns_flow_map_implicit_entry n c {- 144 -}    = DePair
-                                            ^ ( ns_flow_map_yaml_key_entry n c
-                                              / c_ns_flow_map_empty_key_entry n c
-                                              / c_ns_flow_map_json_key_entry n c )
+                                            ^ ( c_ns_flow_map_json_key_entry n c
+                                              / ns_flow_map_yaml_key_entry n c
+                                              / c_ns_flow_map_empty_key_entry n c )
```

Comment:
so you basically reordered the alternatives in rule 144 to try the previously last alternative first?